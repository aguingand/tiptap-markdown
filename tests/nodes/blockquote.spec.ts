import { test, expect } from "vitest";
import { parse, serialize } from "../utils";


test('parse markdown', () => {
    expect(parse('> example')).toMatchSnapshot();
});
test('parse html', () => {
    expect(parse('<blockquote>example</blockquote>')).toMatchSnapshot();
});
test('serialize', () => {
    expect(serialize('<blockquote>example</blockquote>')).toEqual('> example\n')
});
