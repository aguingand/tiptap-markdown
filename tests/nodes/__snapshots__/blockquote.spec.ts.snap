// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`parse html 1`] = `
[
  {
    "content": [
      {
        "content": [
          {
            "text": "example",
            "type": "text",
          },
        ],
        "type": "paragraph",
      },
    ],
    "type": "blockquote",
  },
]
`;

exports[`parse markdown 1`] = `
[
  {
    "content": [
      {
        "content": [
          {
            "text": "example",
            "type": "text",
          },
        ],
        "type": "paragraph",
      },
    ],
    "type": "blockquote",
  },
]
`;
